runescape-client/src/main/java/ItemComposition.java

```diff
@@ -246,16 +246,19 @@
    @ObfuscatedGetter(
       intValue = -617679473
    )
-   @Export("int1")
-   public int int1;
+   @Export("teamId")
+   public int teamId;
    @ObfuscatedName("ba")
    @ObfuscatedSignature(
       signature = "Lhz;"
    )
-   IterableHashTable field3683;
+
+   @Export("params")
+   IterableHashTable params;
+
    @ObfuscatedName("bl")
-   @Export("isTradable")
-   public boolean isTradable;
+   @Export("unnoted")
+   public boolean unnoted;
```

Comment:
This isn't correct, it's a flag saying if the item is searchable in things like the search item dialog.

Reply:
@Joshua-F Just tested it and yes that does seem to be right. Flagged true means item is searchable ![https://i.imgur.com/RMNR5Iw.png](https://i.imgur.com/RMNR5Iw.png)

```java
// Staff of light
case: 22296:
	def.femaleModel = 34507;
	def.inventoryActions = new String[] {"null", "Wield", "null", "null", "null"};
	def.inventoryModel = 34518;
	def.maleModel = 34508;
	def.members = true;
	def.modelPitch = 364;
	def.modelRoll = 1508;
	def.note = 22297;
	def.offsetX = 1;
	def.offsetY = 21;
	def.placeHolderId = 22298;
	def.price = 1000005;
	def.unnoted = true;
	def.zoom = 1853;
break;
```

flag false not searchable

![https://i.imgur.com/Vjg1X2y.png](https://i.imgur.com/Vjg1X2y.png)

```java
// Starter staff
case: 22335:
	def.colorToReplace = new short[] {7073};
	def.colorToReplaceWith = new short[] {11065};
	def.femaleModel = 534;
	def.femaleOffset = 6;
	def.inventoryActions = new String[] {"null", "Wield", "Check", "null", "Destroy"};
	def.inventoryModel = 2648;
	def.maleModel = 534;
	def.modelPitch = 790;
	def.modelRoll = 1194;
	def.modelYaw = 117;
	def.offsetX = -3;
	def.params = {451=Check};
	def.placeHolderId = 22336;
	def.price = 500;
	def.zoom = 1158;
break;
```

both items are tradable and unnoted